install.sh

```diff
@@ -36,8 +36,9 @@ cp "$BATS_ROOT"/man/bats.7 "$PREFIX"/share/man/man7
 
 # fix broken symbolic link file
 if [ ! -L "$PREFIX"/bin/bats ]; then
-    rm -f "$PREFIX"/bin/bats
-    ln -s "$PREFIX"/libexec/bats "$PREFIX"/bin/bats
+    dir=$(readlink -e "$PREFIX")
```

Comment:
Sorry for the delay, but confirmed locally this approach works. Just one thing: please enclose the `$(readlink -e "$PREFIX")` in double-quotes, in case the result includes spaces.